code/mob/living/critter/sawfly.dm

```diff
@@ -115,9 +104,20 @@ This file is the critter itself, and all the custom procs it needs in order to f
 		else
 			src.foldself()
 
-	Cross(atom/movable/mover) //code that ensures projectiles hit them when they're alive, but won't when they're dead
-		if(istype(mover, /obj/projectile))
+	Cross(atom/movable/mover)
+
+		//since sawflies have a density of 1 and will auto-set intent to harm, we have to recode tile-sharing
+		if(istype(mover, /mob/living))
```

Comment:
not all mob/livings are tangible, for example this will stop AI eyes moving through them

Reply:
should be fixed now, added a parent call to let other functions decide